scripts/Inelastic/Direct/ReductionHelpers.py

```diff
@@ -228,10 +228,12 @@ def build_subst_dictionary(synonims_list=None) :
         if len(keys) < 2 :
             raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present")
         if len(keys[0]) == 0:
-            raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present, but the first key is empty")
+            raise AttributeError("The pairs in the synonyms fields have to have form key1=key2=key3 with at least two values present, "
```

Comment:
The `max-line-length` set in `flake8` and what was set in `pylint` is `140`. What should it be set to make it look good in github? and do we care?